src/ontology/cl-edit.owl

```diff
@@ -9500,6 +9500,9 @@ SubClassOf(obo:CL_0000705 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000097
 # Class: obo:CL_0000706 (choroid plexus epithelial cell)
 
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:add") Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "JB:jb") Annotation(oboInOwl:hasDbXref "PMID:9550134") obo:IAO_0000115 obo:CL_0000706 "Specialized ependymal cell that produces the cerebrospinal fluid from the blood and secretes it into the lumen of the brain and spinal chord.")
+AnnotationAssertion(obo:RO_0002175 obo:CL_0000706 obo:NCBITaxon_9606)
```

Comment:
Is this also another addition related to the `human_reference_atlas` subset?

Reply:
Removed.

Reply:
Thanks!